runtime/interpreter/interpreter.go

```diff
@@ -1668,6 +1674,29 @@ func (interpreter *Interpreter) compositeDestructorFunction(
 	}
 }
 
+func (interpreter *Interpreter) interfaceFunctions(
```

Comment:
@turbolent I think this would require a flag to check if it is interface / composite function then because interface on checking if interface function has an implementation. 

Though I am not 100% sure if that check is required.

Reply:
I see, compared to `compositeFunction`, `interfaceFunctions` checks if the declaration has statements, makes sense 👍 
Good idea to only allocate the map if it is needed 👌